### PR TITLE
Remove selenium-standalone; update Selenium.WebDriver

### PR DESCRIPTION
**Pull request by SteveSandersonMS**
Eliminates the Node-based selenium-standalone, and switches over to the newer [Selenium Manager](https://www.selenium.dev/blog/2022/introducing-selenium-manager/) which has been built into Selenium since about a year ago.

In theory this should mean it takes care of acquiring an up-to-date version of ChromeDriver on its own. It also means we can remove a lot of code that was interacting with selenium-standalone to start and stop it.

I've also taken the opportunity to remove the redundant Sauce-related code from Components E2E, but left it alone for SignalR tests since it still seems to be used there.

Works fine locally, but let's see how it goes in CI... **Update:** It's passed all the E2E tests 3 times in a row (except the one I deliberately made fail to check that failures are reported) so it does seem good.

### CODE REVIEW

File: src/Shared/E2ETesting/selenium-config.json
Comment (SteveSandersonMS) [inline src/Shared/E2ETesting/selenium-config.json:4]: In theory we no longer have to configure this or keep updating it manually.

We **do** still have to update `SeleniumWebDriverVersion` in `Versions.props` on a regular cadence, since even though it can acquire newer ChromeDriver versions on its own, it still has to be an up-to-date `Selenium.WebDriver` NuGet package in order to support those newer versions.

**Comment by SteveSandersonMS** [comment]
@wtgodbe I think this should go into `release/8.0` because otherwise soon we won't be able to run the E2E tests for servicing 8.0, as Chrome will keep updating. Once it merges into `release/8.0` it can be ported to `main` too.

Does that sound OK to you? Do we need shiproom approval to make a build-infra-only change?

**Comment by javiercn** [comment]
@SteveSandersonMS thanks steve!

This is great! I was running into issues with the E2E tests crashing in the past in my other update attempts.

**Review by javiercn — APPROVED** [review]

**Comment by javiercn** [comment]
Build infrastructure changes are tell-mode only

**Comment by SteveSandersonMS** [comment]
Thanks @javiercn.

The CI failure here is unrelated to this change, and it's tell-mode, so merging.

**Pull request merged**
